client/my-sites/stats/stats-purchase/stats-purchase-svg.tsx

```diff
@@ -22,21 +22,21 @@ const StatsPurchaseSVG = ( {
 					<rect width="375.864" height="273" x="40" y="55" fill="#fff" rx="12.27" />
 					<path
```

Comment:
> <img width="417" alt="image" src="https://github.com/Automattic/wp-calypso/assets/4044428/760619cb-7f48-4912-b771-45b5c802a182">
There's some text legibility issue due to the blue circle landing perfectly within the `o' within `for` 😆 

Maybe we could solve this by adding a subtle blur backdrop via `backdrop-filter: blur(2px);`?

<img width="404" alt="image" src="https://github.com/Automattic/wp-calypso/assets/4044428/b86fbef8-719b-434c-91b9-ce2f04efb052">

We could also try a partially transparent background on the paragraph element.

Reply:
Thank you for the hint. I added the blur and moved the test 23px up.